### ISSUE TITLE
'class' is a reserved attribute and can't be used as a prop

### ISSUE THREAD

**Issue by wopian** [TahaSh/vue-paginate#51]
Vue produces this warning on >2.3.0 if you're trying to use reserved keywords

```
vue.runtime.esm.js?a427:425 [Vue warn]: "class" is a reserved attribute and cannot be used as component prop.

found in

---> <Paginate>
       <Profile> at client\components\user\Profile.vue
         <User> at client\components\User.vue
           <App> at client\App.vue
             <Root>
```

Hopefully this isn't dead as per #50 😢

**Comment by TahaSh** [comment]
@wopian I'll include a fix for this in my next release (which should be out soon). Thanks for pointing it out. And don't worry, it isn't dead 😉  – sorry for my long absence.

**Comment by callumacrae** [comment]
I'm getting this too 😢 would be great to get a release soon! Let me know if there's anything simple I can help with (although I'm not familiar with the v3 code)

Is the code pushed to github at all so I can use an unstable version until released? This is basically blocking an entire project right now because it's part of the Vue 2 update, so we would have 1000 merge conflicts if we tried to do anything else 😁

**Comment by callumacrae** [comment]
just noticed fix in https://github.com/TahaSh/vue-paginate/pull/52

**Comment by M7Dev** [comment]
Having the same issue when passing an object. An array however, does not seem to give any error message.

**Comment by TahaSh** [comment]
This issue has been fixed in v3.5.0. Thanks @callumacrae for your suggestion to remove it altogether :).
